src/server/game/Spells/SpellEffects.cpp

```diff
@@ -61,13 +61,18 @@
 #include "Vehicle.h"
 #include "World.h"
 #include "WorldPacket.h"
+#include <chrono>
+#include <iostream>
+#include <ctime>
 
  // TODO: this import is not necessary for compilation and marked as unused by the IDE
 //  however, for some reasons removing it would cause a damn linking issue
 //  there is probably some underlying problem with imports which should properly addressed
 //  see: https://github.com/azerothcore/azerothcore-wotlk/issues/9766
 #include "GridNotifiersImpl.h"
 
+using namespace std;
```

Comment:
Remove the namespace please :D

Reply:
hopefully it compiles. I put the std in STuD and all I need is U.

Reply:
Probably it won't, you'll need to add std:: before the things :P

Reply:
it compiles :D